addons/account/models/chart_template.py

```diff
@@ -201,6 +201,9 @@ def _load(self, template_code, company, install_demo):
                     self.env[model].sudo().with_context(active_test=False).search([('company_id', 'child_of', company.id)]).with_context({MODULE_UNINSTALL_FLAG: True}).unlink()
 
         data = self._get_chart_template_data(template_code)
+        if not data.get('template_data'):
+            return
```

Comment:
It's because it's a temporary fix, i'm waiting for william to come back because the KeyError: 'template_data' is a general issue and i'm not sure how to fix this. 
The current fix works but it's more fixing the problem and not the cause of it 🤔 

Maybe just a user error but then the feature of putting the country on the partner may be useless